src/content/1.7/modules/testing/basic-checks.md

```diff
@@ -86,3 +86,34 @@ Using cache file ".php_cs.cache".
 
 Checked all files in 0.085 seconds, 12.000 MB memory used
 ```
+
+## License headers
+
+When the extension is open-source, sharing the license is one important task.
+
+[Header-Stamp](https://github.com/PrestaShopCorp/header-stamp) is a tool that applies a header on all compatible files (PHP, JS...).
+In these headers, details can be found about the author, the license, the original year of publication.
+Everytime there is a change to make, one command would apply it to all your files.
+
+This tool is part of `prestashop/php-dev-tools` available on [Packagist](https://packagist.org/packages/prestashop/php-dev-tools), which can be required via composer.
+
+As an example, the license AFL is going to be applied on a PrestaShop native module. There is a AFL template existing in the package that can be used,
+and there are a few folders to ignore while these changes are applied (`vendor` because libraries have their own license, `test` and `_dev` because they
+won't be part of the release package).
```

Comment:
I reworded this a bit, then removed the "native" part, because we wouldn't ignore tests and _dev on a native module.
```suggestion
### Example

In the following example, the AFL license is going to be applied on a module. 

The Header-Stamp package already includes an AFL template, so we can use that one. We also have a few folders that we will want to ignore: `vendor` (because libraries have their own license), `test`, and `_dev` (because they won't be part of the release package).
```